packages/app/src/Element/NoteFooter.tsx

```diff
@@ -209,6 +209,13 @@ export default function NoteFooter(props: NoteFooterProps) {
   function menuItems() {
     return (
       <>
+        <div className="close-menu-container">
```

Comment:
adding a div wrapper here prevents the newly added menu item from altering the existing menu's styling/appearance.